src/scripts/sync_subclassof.py

```diff
@@ -308,7 +308,7 @@ def sync_subclassof(
     #  after. It could be that I invalidated the cache somehow without realizing it. Rerunning cache fixed it though.
     missing_ancestor_rels = rels_indirect_mondo_mondo.union(rels_direct_mondo_mondo).difference(ancestors_mondo_mondo)
     if missing_ancestor_rels:
-        rels_for_printing =  [rel for rel in missing_ancestor_rels[:5]]
+        rels_for_printing =  [rel for rel in list(missing_ancestor_rels)[:5]]
```

Comment:
Curious, why is it limited to the first 5?

Reply:
There could be 10000s..